index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./src/style.css">
+    <title>Apple</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div><i class="fab fa-apple"></i></div>
+            <div>Store</div>
+            <div>Mac</div>
+            <div>iPod</div>
+            <div>iPhone</div>
+            <div>iPad</div>
+            <div>iTunes</div>
+            <div>Support</div>
+            <div>
+                <form>
+                    <button type="submit"><i class="fas fa-search"></i></button>
+                    <input />
+                </form>
+            </div>
+        </nav>
+        <div class="header-text">
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See hoe people are using <br> iPad in their unique ways to add to the
+                world's
+                story</h2>
+            <h3>Watch the "Your verse" film and explore the stories <i class="far fa-play-circle"></i></h3>
+        </div>
+    </header>
+    <main>
+        <div class="product-grid">
+            <div class="fives"></div>
+            <div class="fivec"></div>
+            <div class="mac"></div>
+            <div class="supplier"></div>
```

Comment:
- [ ] Images should be links. Each image is a link that redirects to another page in the site. The elements should mimic the original website functionality.